src/components/Icon/index.js

```diff
@@ -426,24 +426,33 @@ export const Glyph = ({ handle }) => {
   }
 }
 
-export const Icon = ({ size = 32, glyph, ...props }) => (
-  <SvgWrapper size={size} className="icon" {...props}>
-    <InlineSvg
-      fillRule="evenodd"
-      clipRule="evenodd"
-      strokeLinejoin="round"
-      strokeMiterlimit="1.414"
-      xmlns="http://www.w3.org/2000/svg"
-      role="presentation"
-      focusable="false"
-      viewBox="0 0 32 32"
-      preserveAspectRatio="xMidYMid meet"
-      fit
-    >
-      <Glyph handle={glyph} />
-    </InlineSvg>
-  </SvgWrapper>
-)
+export const Icon = ({ size = 32, glyph, ...props }) => {
+  React.useEffect(() => {
+    process.env.NODE_ENV === 'development' &&
```

Comment:
Shouldn't we remove the whole effect if not dev, since it's empty for not dev 🤔

Reply:
Yeah but you can’t run a hook conditionally. Think it’s fine until we remove the Icon component. Otherwise I might consider removing that warning, lol 😅